pkg/graph/schema.graphql

```diff
@@ -399,7 +398,6 @@ https://gqlgen.com/reference/changesets/
 """
 input PlanDiscussionChanges @goModel(model: "map[string]interface{}") {
   content: String
-  status: DiscussionStatus

```

Comment:
Discussion status still exists in the `GO` struct and in the database. This should probably be removed like it was for the reply.